component/toolbox/Dockerfile

```diff
@@ -6,7 +6,8 @@ RUN set -eux; \
         https://s3.amazonaws.com/session-manager-downloads/plugin/latest/linux_${arch}/session-manager-plugin.rpm; \
     yum install -y jq
 
-COPY ./scripts/* /usr/local/bin/si/
+COPY ./scripts/ /usr/local/bin/si/
```

Comment:
Looks like a mistake, but this glob was flattening the structure of the copy, I want it to retain the structure of the dirs for the supporting-funcs so that I can accurately import them when executing